<launch>

    <!-- enable point-cloud. This overrides enable_pc setting in config/params.yaml -->
    <arg name="enable_pc" default="0"/>

    <node pkg="ti_sde" type="sde" name="sde" output="screen" args="" required="true">
        <!-- Configuration file for the openVX CNN graph -->
        <rosparam file="$(find ti_sde)/config/params.yaml" subst_value="true" />
        <rosparam subst_value="true">
            enable_pc: $(arg enable_pc)
        </rosparam>

        <!-- Left input topic name to subscribe to -->
        <param name = "left_input_topic"  value = "camera/left/image_raw"/>

        <!-- Right input topic name to subscribe to -->
        <param name = "right_input_topic" value = "camera/right/image_raw"/>

        <!-- Right camera parameter topic name to subscribe to -->
        <param name = "camera_info_topic" value = "camera/right/camera_info"/>

        <!-- Ouput raw dispairty topic to publish to -->
        <param name = "disparity_topic"    value = "camera/disparity/raw"/>

        <!-- Ouput point cloud topic to publish to -->
        <param name = "point_cloud_topic"    value = "point_cloud"/>
    </node>

</launch>
